src/Entity/User.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\UserRepository;
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Timestampable\Traits\TimestampableEntity;
+
+#[ORM\Entity(repositoryClass: UserRepository::class)]
+#[ORM\Table(name: '`user`')]
+class User
+{
+    use TimestampableEntity;
+
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column]
+    private ?int $id = null;
+
+    #[ORM\Column(length: 255)]
+    private ?string $name = null;
+
+    #[ORM\Column(length: 255)]
+    private ?string $email = null;
+
+    #[ORM\Column(length: 255)]
+    private ?string $mobileNumber = null;
+
+    #[ORM\Column]
+    private ?int $employeeId = null;
+
+    #[ORM\Column(nullable: true)]
+    private ?int $approvedBy = null;
+
+    #[ORM\Column(nullable: true)]
+    private ?bool $status = null;
+
+    public function getId(): ?int
+    {
+        return $this->id;
+    }
+
+    public function getName(): ?string
+    {
+        return $this->name;
+    }
+
+    public function setName(string $name): static
+    {
+        $this->name = $name;
+
+        return $this;
+    }
+
+    public function getEmail(): ?string
+    {
+        return $this->email;
+    }
+
+    public function setEmail(string $email): static
+    {
+        $this->email = $email;
+
+        return $this;
+    }
+
+    public function getMobileNumber(): ?string
+    {
+        return $this->mobileNumber;
+    }
+
+    public function setMobileNumber(string $mobileNumber): static
+    {
+        $this->mobileNumber = $mobileNumber;
+
+        return $this;
+    }
+
+    public function getEmployeeId(): ?int
+    {
+        return $this->employeeId;
+    }
+
+    public function setEmployeeId(int $employeeId): static
+    {
+        $this->employeeId = $employeeId;
+
+        return $this;
+    }
+
+    public function getApprovedBy(): ?int
+    {
+        return $this->approvedBy;
+    }
+
+    public function setApprovedBy(?int $approvedBy): static
+    {
+        $this->approvedBy = $approvedBy;
+
+        return $this;
+    }
+
+    public function isStatus(): ?bool
+    {
+        return $this->status;
+    }
+
+    public function setStatus(?bool $status): static
+    {
+        $this->status = $status;
+
+        return $this;
+    }
```

Comment:
We need to include two additional fields: 'created_at' and 'updated_at'.

Reply:
we are using TimestampableEntity which will automatically insert those columns. Please check migration file.